### ISSUE TITLE
Build Global Nav

### ISSUE THREAD

**Issue by cavis** [PRX/feeder.prx.org#504]
After #503.  Build out the [global nav bar at the top](https://www.figma.com/proto/Yewv83Q8btHcaOBWCtGgKs/PRX-Dovetail-with-Navigation?page-id=80%3A2947&node-id=2934%3A80025&viewport=798%2C-3032%2C0.25&scaling=min-zoom&starting-point-node-id=80%3A3420&hide-ui=1), which is (fairly) static, and will eventually be used across apps.

![image](https://user-images.githubusercontent.com/1410587/212186885-9a9b274f-0d8b-470c-9e06-e133c325cccd.png)

- [ ] Build it (leave out "Collections", which we don't have yet)
- [x] Make it _fairly_ bootstrap-only-ish. As far as is reasonable. Since we'll be sharing this across apps.
- [x] Make it responsive
- [x] Build the help dropdown
- [ ] Build the user-avatar + dropdown
  - This data will have to be pulled from the `current_user`... which is a `prx_auth-rails` thing
  - We may need some additional helpers in that gem to get all these apps/links/etc
  - But if we don't have that yet, fake it!  We'll fill in the real data later.
- [ ] Don't hardcode links, so this works in staging and prod.  They should come from ID, I think.
- [ ] I18n the text

**Comment by cavis** [comment]
Note: some of this is already done by @brandonhundt.  But we need to tie in the actual ID/User data, once authentication is built out.
